packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -472,11 +472,7 @@ export const PersonalDetailsForm = ({
         { text: localize('Miss'), value: 'Miss' },
     ];
 
-    const {
-        form_initial_values: { ...form_initial_values },
-        api_error,
-        show_form,
-    } = rest_state;
+    const { form_initial_values, api_error, show_form } = rest_state;
```

Comment:
@balakrishna-binary Could u please check this
we had an issue with destructing form_initial_values! so I removed them!
It seems that it is working properly but I asked QA to check it too

Reply:
I think it shouldn't be a problem.